File: Iniciante/1072.js
lines = lines.map((e) => Number(e));

let lim = lines.shift(), contIn = 0, contOut = 0;

for (let i = 0; i < lim; i++) {
  if (lines[i] >= 10 && lines[i] <= 20) {
    contIn++;
  } else {
    contOut++;
  }
}

console.log(`${contIn} in\n${contOut} out`);

File: Iniciante/1072.py
lim = int(input())
contIn = 0 
contOut = 0

for i in range(lim):
  valor = int(input())
  if valor >= 10 and valor <= 20:
    contIn += 1
  else:
    contOut += 1

print("{} in\n{} out".format(contIn, contOut))
